### PR TITLE
Hello world microverse

### PR DESCRIPTION
**Pull request by iwillteixeira**
### Hello world Microverse

In this Hello world project we did:

- Added index.html with h1 tag.
- Added style.css and its link to index.html

### CODE REVIEW

File: .github/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Comment (GzimAsani) [inline .github/linters.yml]: - [x] Please add linters properly, this particular linter should be as `.github/workflows` and then .linters.yml
- [x] Also add `.hintrc`

_**If you need a refresher for linters you can visit [LINTERS](https://github.com/microverseinc/linters-config/tree/master/html-css)**_

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# .gitignore
```
Comment (GzimAsani) [inline .gitignore:1]: - [x] Files that are not part of the project please add to the `.gitignore` files such as folder `hint-report`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```
Comment (GzimAsani) [inline index.html:1]: - [x] Update your readme and explaining your project. Currently, you have just copied/pasted the Microverse template. Please change the content so it matches your project description.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @iwillteixeira                
Good job so far! Your very first project looks great :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- PR has a detailed description :+1: 
- Gitflow is used :clap: 

## Required Changes ♻️ 
- [x] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/linters.yml (hunk 1 above)
Comment (iwillteixeira) [reply .github/linters.yml]: Done, linters.yml was moved for the right place, and .hintrc was created on the root file.

File: .gitignore (hunk 2 above)
Comment (iwillteixeira) [reply .gitignore:1]: Added hint-report on .gitignore.

File: index.html (hunk 3 above)
Comment (iwillteixeira) [reply index.html:1]: README.md was updated.

**Review by mikethreels — APPROVED** [review]
Hi Will, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**